src/pages/workspace/workflows/WorkspaceWorkflowsPage.tsx

```diff
@@ -0,0 +1,167 @@
+import * as Policy from '@userActions/Policy';
+import CONST from '@src/CONST';
+import type SCREENS from '@src/SCREENS';
+import ToggleSettingOptionRow from './ToggleSettingsOptionRow';
+import type {ToggleSettingOptionRowProps} from './ToggleSettingsOptionRow';
+
+type WorkspaceWorkflowsPageProps = WithPolicyProps & StackScreenProps<CentralPaneNavigatorParamList, typeof SCREENS.WORKSPACE.WORKFLOWS>;
+
+function WorkspaceWorkflowsPage({policy, route}: WorkspaceWorkflowsPageProps) {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+    const StyleUtils = useStyleUtils();
+    const {isSmallScreenWidth} = useWindowDimensions();
+    const {isOffline} = useNetwork();
+
+    const [policyOwnerDisplayName, setPolicyOwnerDisplayName] = useState('');
+
+    useEffect(() => {
+        if (!policy?.ownerAccountID) {
+            return;
+        }
+        const ownerPersonalDetails = ReportUtils.getDisplayNamesWithTooltips(OptionsListUtils.getPersonalDetailsForAccountIDs([policy.ownerAccountID], CONST.EMPTY_OBJECT), false);
+        if (ownerPersonalDetails.length > 0 && ownerPersonalDetails[0].displayName) {
+            setPolicyOwnerDisplayName(ownerPersonalDetails[0].displayName);
+        }
+    }, [policy]);
+
+    const containerStyle = useMemo(() => [styles.ph8, styles.mhn8, styles.ml11, styles.pv3, styles.pr0, styles.pl4, styles.mr0, styles.widthAuto, styles.mt4], [styles]);
+
+    const items: ToggleSettingOptionRowProps[] = useMemo(
+        () => [
+            {
+                icon: Illustrations.ReceiptEnvelope,
+                title: translate('workflowsPage.delaySubmissionTitle'),
+                subtitle: translate('workflowsPage.delaySubmissionDescription'),
+                onToggle: (isEnabled: boolean) => {
+                    Policy.setWorkspaceAutoReporting(route.params.policyID, isEnabled);
+                },
+                subMenuItems: (
+                    <MenuItem
+                        title={translate('workflowsPage.submissionFrequency')}
+                        titleStyle={styles.textLabelSupportingNormal}
+                        descriptionTextStyle={isOffline ? StyleUtils.getWorkspaceWorkflowsOfflineDescriptionStyle(styles.textNormalThemeText) : styles.textNormalThemeText}
+                        // onPress={() => Navigation.navigate(ROUTES.WORKSPACE_WORKFLOWS_AUTOREPORTING_FREQUENCY).getRoute(route.params.policyID))}
+                        // TODO will be done in https://github.com/Expensify/Expensify/issues/368332
+                        description={translate('workflowsPage.weeklyFrequency')}
+                        shouldShowRightIcon
+                        wrapperStyle={containerStyle}
+                        // wrapperStyle={workflowsStyles.subMenuContainer}
+                        hoverAndPressStyle={[styles.mr0, styles.br2]}
+                    />
+                ),
+                hasBeenToggled: policy?.harvesting?.enabled ?? false,
```

Comment:
NAB `hasBeenToggled` confuses me as a name, as I expect an action to have just been taken. Maybe `isEnabled` or `isActive` would be better

Reply:
I don't think `isEnabled` would be a good choice as it would be confusing since we already have a `isEnabled` property in `ToggleSettingOptionRow`

Reply:
Yea, maybe `isActive` then